### PR TITLE
Move the Medicine information from the General Guidance page into its own page (FHIR-45165)

### PR DESCRIPTION
**Pull request by dbojicic**
Changes to address https://jira.hl7.org/browse/FHIR-45165. 

Move the Medicine information from the General Guidance page into its own page: 
- added new Medicine Information page
- updated home page > How to read this guide
- updated intro notes in AU Core Medication and AU Core MedicationRequest to link the new page
- updated general-guidance.md
- updated change log

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-medication-intro.md [hunk 1]
```diff
@@ -18,5 +18,5 @@ A resource conforming to this profile is conformant to:
 Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://build.fhir.org/ig/HL7/fhir-ipa), [International Patient Summary](http://build.fhir.org/ig/HL7/fhir-ips), or [US Core](http://hl7.org/fhir/us/core) **MAY NOT** conform to AU Core.
 
 ### Profile specific implementation guidance
-- See the [Medicine information](general-guidance.html#medicine-information) section for guidance on constructing a resource and the use of medicines terminology
+- See the [Medicine Information](medicine-information.html) section for guidance on constructing a resource and the use of medicines terminology
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-medication-intro.md:21]: say "page" not "section".

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 
 ### Profile specific implementation guidance
-- See the [Medicine information](general-guidance.html#medicine-information) section for guidance 
+- See the [Medicine Information](medicine-information.html) section for guidance
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md:21]: say "page" not "section".

File: input/pagecontent/changes.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 
 - Added Security and Privacy page [FHIR-45067](https://jira.hl7.org/browse/FHIR-45067).
+- Added Medicine Information page [FHIR-45165](https://jira.hl7.org/browse/FHIR-45165).
```
Comment (dt-r) [inline input/pagecontent/changes.md:11]: quality this to be clear the information from general guidance on medicine information was moved to it's own page.

File: input/pagecontent/index.md [hunk 4]
```diff
@@ -118,6 +118,7 @@ This guide is divided into several pages which are listed at the top of each pag
   - [General Requirements](general-requirements.html): This page defines requirements common to all actors and profiles used in this guide including how CapabilityStatements are used to claim conformance. This page defines the expectations for mandatory and must support elements in AU Core Profiles.
 - [Guidance](guidance.html): This set of pages lists the guidance for this guide.
   - [General Guidance](general-guidance.html): This page provides guidance on using the profiles defined in this guide.
+  - [Medicine Information](medicine-information.html): This page provides guidance on the use of medications related profiles defined in this guide.
```
Comment (dt-r) [inline input/pagecontent/index.md:121]: Change this slightly "This page provides guidance on constructing medications related resources."

**Review by dt-r — CHANGES_REQUESTED** [review]
Few comments.

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md (hunk 2 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md]: Fixed.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Fixed.

File: input/pagecontent/index.md (hunk 4 above)
Comment (dbojicic) [reply input/pagecontent/index.md]: Fixed.

**Review by dt-r — APPROVED** [review]

**Pull request merged**